cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/MonitorHandlerTest.java

```diff
@@ -102,23 +102,22 @@ public void testInstances() throws Exception {
 
   @Test
   public void testRestartInstances() throws Exception {
-    String path = String.format("system/services/%s/instances/restart", Constants.Service.APP_FABRIC_HTTP);
-    HttpURLConnection urlConn = openURL(path, HttpMethod.PUT);
-
-    Assert.assertEquals(HttpResponseStatus.OK.getCode(), urlConn.getResponseCode());
+    String path = String.format("%s/system/services/%s/restart", Constants.Gateway.API_VERSION_3,
```

Comment:
is there a test case for restarting a single instance, and then getting the status of that restart?